README.md

```diff
@@ -1,44 +1,53 @@
 # JustBar
+A HMOS library to add a Bar.
```

Comment:
A HMOS library to add a Bottom Navigation Bar. - Please check this and modify

Reply:
Done